main.go

```diff
@@ -83,3 +117,19 @@ func VerboseLog(msg string) {
 		fmt.Println("*", msg)
 	}
 }
+
+func WriteStringProperty(b *bytes.Buffer, name string, value string) {
+	if value != "" {
+		WriteProperty(b, name, value)
+	}
+}
+
+func WriteUInt8Property(b *bytes.Buffer, name string, value uint8) {
+	if value != 0 {
```

Comment:
I think there is a need, because I can add a header and call it "correlation_id" and it'll have different value than the property with the same name.